community-operators/mongodb-enterprise-upstream/v0.3.2/mongodb.v0.3.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,311 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
These files do no need to be versioned. The directory structure for this operator should be as follows:
```
community-operators/mongodb-enterprise/*
```

Reply:
If this is meant for upstream the files should be placed in `upstream-community-operators/mongodb-enterprise/*`